style.css

```diff
@@ -0,0 +1,877 @@
+@import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Nabla&family=Poppins&display=swap');
+
+body {
+  padding: 0%;
+  margin: 0%;
+  background-color: #f7f7f9;
+}
+
+/* minimum mobile size 375px */
+
+@media screen and (min-width: 375px) {
+  /* header style */
+
+  #header {
+    height: 7vh;
+    position: top;
+    display: flex;
+    padding: 3px;
+    align-items: center;
+    background-color: #ffff;
+  }
+
+  #navigation {
+    position: relative;
+    width: 100%;
+    padding: 0%;
+  }
+
+  #navigation ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    text-decoration: none;
+    align-items: center;
+    list-style: none;
+    text-align: center;
+  }
+
+  #logo {
+    width: 30%;
+    height: 20px;
+    text-decoration: none;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
```

Comment:
- Great job for using the exact font family and weight :+1: however `font-weight:700 `hasn't been imported, so please import it to have the exact look as the Figma design.check this for [more info](https://stackoverflow.com/questions/7256065/specifying-style-and-weight-for-google-fonts) :+1: